sound/effect.h

```diff
@@ -394,6 +397,11 @@ class Effect {
   }
 
   Effect* next_;
+  Effect* current_;
+  
+  // All files must start with this prefix. Public so other functions can use it
+  const char* name_ = "";
```

Comment:
We don't need this name.
If we have a pointer to the Effect, we can just use that to identify what is being played.

Reply:
This is were I struggle I have to confess. I can see the Effect* for current_ and next_ int the effecht.h. 
Currently I 'nick' the effectname and effect2ndname from the PlayOnce() and initialize them there.
If you could give me a hint how to get an Effect* directly to the currently playing wav for the effect, that would help a lot.